js/src/modal.js

```diff
@@ -139,13 +139,17 @@ class Modal extends BaseComponent {
   }
 
   dispose() {
-    EventHandler.off(window, EVENT_KEY)
-    EventHandler.off(this._dialog, EVENT_KEY)
+    this._element.addEventListener(EVENT_HIDDEN, () => {
```

Comment:
```suggestion
    EventHandler.on(this._element, EVENT_HIDDEN, () => {
```
 
 
 try this syntax to be aligned with the rest codebase ;wink